tms_objects2.erb

```diff
@@ -529,9 +529,20 @@ xsi:schemaLocation="http://www.lido-schema.org http://www.lido-schema.org/schema
               <lido:conceptID lido:source="ATT" lido:type="local">300157782</lido:conceptID>
               <lido:term>acquisition</lido:term>
             </lido:eventType>
+            <lido:eventActor>
+              <lido:displayActorInRole><%= Owners[0]["DisplayName"] %></lido:displayActorInRole>
+        <% unless Owners[0]["DisplayBio"].nil? %>
+              <lido:displayActorInRole lido:label="New bio line"><%= Owners[0]["DisplayBio"] %></lido:displayActorInRole>
```

Comment:
DisplayBio is a new field, you probably created the Owner view after this was added. so need to refresh the view with that field:

https://github.com/ycba-cia/coboat-github/blob/main/bac-prd-db1/config/LIDOTMSretrieve.cbcs#L81 

Also will probably have to add an actual DisplayBio data to LIDOTMS for the test, we can talk about that after you fix the view.